lib/web_task_runner/version.rb

```diff
@@ -1,3 +1,3 @@
 class WebTaskRunnerVersion
-  VERSION = "0.0.5"
+  VERSION = "0.0.5.1"
```

Comment:
I guess I'll just bump it to a six.
RubyGems seems not to support version numbers constructed by more then three parts (?).